robocup_msgs/Node new_node
---
bool success
